L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -513,6 +518,7 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
   bool receiveTau = true;
   bool receiveJet = true;
   bool receiveEtSums = true;
+  bool receiveEtSumsZdc = true;
```

Comment:
Would it make sense to set this to `false` if `m_sumZdcInputTag` is empty ?

Reply:
I think in this instance we want to leave it as is if only to conform with the rest of these collections - I followed the variable forward to GlobalBoard.cc where it is finally used in a non-passthru way.

https://github.com/cfmcginn/cmssw/blob/l1tZDCEmulator_CMSSW13_3_X/L1Trigger/L1TGlobal/src/GlobalBoard.cc#L322

If receiveEtSumsZdc is true but the collection is empty it just logs a warning and doesn't push_back; this conforms with the behavior of all the other collections so I'm assuming this is preferred (but I'm just inferring that)

No change made for this in latest commit

Reply:
I'm fine with keeping it as is for this PR, bit I think this (and maybe the similar cases) should be cleaned up later.